tests/phpunit/integration/Modules/Analytics_4/Resource_Data_Availability_DateTest.php

```diff
@@ -47,16 +53,22 @@ public function set_up() {
 		$options                               = new Options( $context );
 		$transients                            = new Transients( $context );
 		$this->settings                        = new Settings( $options );
-		$this->resource_data_availability_date = new Resource_Data_Availability_Date( $transients, $this->settings );
+		$this->audience_settings               = new Audience_Settings( $options );
+		$this->resource_data_availability_date = new Resource_Data_Availability_Date( $transients, $this->settings, $options );
 
 		$this->settings->merge(
 			array(
-				'propertyID'         => $this->test_property_id,
+				'propertyID' => $this->test_property_id,
+			)
+		);
+
+		$this->audience_settings->set(
+			array(
 				'availableAudiences' => array(
-					array(
+					$this->test_audience_1 => array(
```

Comment:
This doesn't look correct, `availableAudiences` should be a list of objects, not an associative array.